client/src/assets/stock-history/stock-history-sample.ts

```diff
@@ -0,0 +1,317 @@
+const yahooSampleChart = {
```

Comment:
Was this file created only for testing during development or we do use it somewhere in our app? If it was only for testing we can remove it.

Reply:
yes